GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -13,6 +13,7 @@ class ChatViewModel: EngagementViewModel, ViewModel {
         case fileTapped(LocalFile)
         case downloadTapped(FileDownload)
         case choiceOptionSelected(ChatChoiceCardOption, String)
+        case chatScrolled(Bool)
```

Comment:
This is a bit confusing imo since you're going from `view.chatScrolledToTheEnd` to just `chatScrolled`. Im starting to think `chatScrolledToEnd` (or `chatScrolledToBottom`) should also be renamed to `chatScrolled(reachedBottom: Bool)` everywhere so it reflects what it actually is more clearly.

Reply:
While `chatScrolled(reachedBottom:)` looks and sounds good, it doesn't really work everywhere sadly.

In `UnreadMessagesHandler` it should still be called `isChatScrolledToBottom` because it refers not to an event, but just the boolean value of the result.

In `ChatView` it is a function variable thus parameters cannot be named. Without that semantics are lost (`chatScrolledToBottom((Bool) -> Void)?` vs `chatScrolled((Bool) -> Void)?`